.eslintrc

```diff
@@ -14,8 +14,5 @@
       "jsx": true
     }
   },
-  "plugins": ["react", "json", "jsx-a11y"],
-  "rules": {
-    "prettier/prettier": ["error", {singleQuote: true, trailingComma: "all"}]
-    }
+  "plugins": ["prettier", "json"],
```

Comment:
why is this file changed?

Reply:
@benjaminhobbs React and jsx-a11y plugins are brought in through the air-bnb style guide so we do not need them here. Putting prettier here lets us define its rules in another file our .prettierrc